sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -36,13 +36,7 @@
             {{ item.restrictionCode }}
           </template>
           <template #[`item.restrictionStatus`]="{ item }">
-            <status-chip-restriction
```

Comment:
What happens to a bypass record when the student restriction associated to that is deleted?